site/app/templates/grading/electronic/Status.twig

```diff
@@ -451,24 +460,24 @@
                         {% endif %}
 
                         {% if team_assignment %}
-                            <br />
+                            <br/>
```

Comment:
I am wondering why these `<br />`s are changed to `<br/>`s. They mean the same thing to me. Could I know your considerations?